deptool/src/main/java/io/jmix/dependency/cli/command/ExportCommand.java

```diff
@@ -32,6 +37,12 @@ public void run() {
         log.info("Target directory: {}", Paths.get(targetDirectory).toAbsolutePath().normalize());
         log.info("Gradle user home directory: {}", Paths.get(gradleUserHome).toAbsolutePath().normalize());
 
+        LinkedHashSet<String> exportedDependencies = new LinkedHashSet<>();
```

Comment:
What about using the TreeSet here? In this case we should get the dependencies list sorted